### PR TITLE
Fix date range field - closes #875

### LINKED ISSUE #875: Mods 2 & 4: Bug when entering dates into search filter
If you try to filter by date in the GUI, if you start by entering the month, then go to day, then go to year, things go wonky. Entering year tends to reset the month and day to “- -.” Then, when I type 2019 for year, it gets entered as 0019. To function in spite of this bug, I have to click on year first to enter 2019, which works. Then I go to month to enter a number like 08. Then I enter day. 

image

### PR DESCRIPTION
**Pull request by billoosijok**

### CODE REVIEW

File: app/src/components/Fields/DateRangeField.js [hunk 1]
```diff
@@ -45,11 +38,8 @@ export default function DateRangeField(props) {
 
   function renderSelectValue(value) {
     if (value && (value.to || value.from)) {
-      return `${
-        value.from ? extractTimeAndDateForDisplay(value.from).date : 'All dates'
-      } to ${
-        value.to ? extractTimeAndDateForDisplay(value.to).date : 'All dates'
-      }`;
+      return `${value.from ? formatISODate(value.from) : 'All dates'} to
```
Comment (punkael) [inline app/src/components/Fields/DateRangeField.js:41]: Display the date as DD-MM-YYYY  in this select box so that it matches the date boxes.

Comment (billoosijok) [reply app/src/components/Fields/DateRangeField.js:41]: Good point! Will do that

Comment (billoosijok) [reply app/src/components/Fields/DateRangeField.js:41]: @punkael Actually looks like there is a bug where the dates reset when the popover is dismissed

Comment (billoosijok) [reply app/src/components/Fields/DateRangeField.js:41]: I'll look into it

**Review by punkael — APPROVED** [review]

**Pull request merged**